app/controllers/api/v4/patients_controller.rb

```diff
@@ -15,7 +15,11 @@ def lookup
     trigger_audit_log(patients)
     render(
       json: Oj.dump({
-        patients: patients.map { |patient| Api::V4::PatientLookupTransformer.to_response(patient, retention(patient)) }
+        patients: patients.map do |patient|
+          retention = retention(patient)
+          Statsd.instance.increment("OnlineLookup.#{retention[:type]}", tags: [current_state.name, current_user.id])
```

Comment:
I suspect `current_state` and `current_user` are guaranteed to be present for an authenticated request? If so, :100: